Assets/Scripts/PauseManager.cs

```diff
@@ -0,0 +1,58 @@
+﻿using UnityEngine;
+using UnityEngine.SceneManagement; // シーン遷移用
+
+public class PauseManager : MonoBehaviour
+{
+    public GameObject pauseCanvas; // PauseCanvasをアタッチ
+    public GameObject pauseButton; // PauseButtonをアタッチ
```

Comment:
publicである必要がないので、privateにしよう。

```suggestion
    [SerializeField] private GameObject _pauseCanvas; // PauseCanvasをアタッチ
    [SerializeField] private GameObject _pauseButton; // PauseButtonをアタッチ
```